src/arkhamdb/ArkhamDb.ttslua

```diff
@@ -304,6 +304,12 @@ do
     end
   end
 
+  internal.maybePrint = function(message, playerColor)
```

Comment:
Is there something else in `internal` or why is this not just called `maybePrint`?

Reply:
Internal is used to keep the method from leaking out of the module.  As we get more and more modules the potential for name collisions gets higher.

I'm not sure if I just haven't puzzled out the Lua or TTS handles it badly, but making a method like this local (which is basically the pattern I'm looking for) doesn't work.  I gave up trying to puzzle it out, so this may not be the best way to do it but it works.